packages/mapviewer/src/modules/map/components/openlayers/debug/OpenLayersDeviceOrientationDebugInfo.vue

```diff
@@ -103,9 +103,7 @@ async function copyValue(event, value) {
                     v-for="parameter in parameters"
                     :key="parameter.title"
                 >
-                    <div class="text-decoration-underline fw-bold">
-                        {{ parameter.title }}:
-                    </div>
+                    <div class="text-decoration-underline fw-bold">{{ parameter.title }}:</div>
```

Comment:
I'm confused, it broke many things into multiple line, and for this one, it does the opposite?